out/css/spinner.css

```
@@ -0,0 +1,26 @@
+#cc-spinner {
+  display: inline-block;
+  width: 64px;
+  height: 64px;
+}
+
+#cc-spinner:after {
+  content: " ";
```

This can also be empty.
```suggestion
  content: "";
```

---

please make it empty and not a whitespace